src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig

```diff
@@ -1,63 +1,83 @@
+The third annual No Time to Wait conference shall take place at BFI Southbank sited on the River Thames in London.
 
 <h2>What</h2>
-<p>A FREE two day symposium focused on open media, open standards, and digital audiovisual preservation hosted by <a href="http://www.bfi.org.uk/">British Film Institute</a> and <a href="{{ path('homepage') }}">MediaArea.net</a>.</p>
+<p>A FREE two-day symposium focused on <em>open media</em>, <em>open standards</em>, and <em>digital audiovisual preservation</em> hosted by <a href="http://www.bfi.org.uk/">British Film Institute</a> and <a href="{{ path('homepage') }}">MediaArea.net</a>.</p>
 
 <p>The event will feature presentations and discussion on topics such as:</p>
 
 <ul>
-  <li>(To Be Announced)</li>
+  <li>active open media standardization projects</li>
+  <li>seeking consensus in audiovisual preservation strategy</li>
+  <li>examination of open media use in film and video digitization</li>
+  <li>validation and conformance checking of audiovisual formats</li>
+  <li>integration of open source tools into archival workflow</li>
+  <li>examples of cross-community collaboration and skill-sharing</li>
+  <li>developments in open media</li>
+  <li>and more</li>
 </ul>
 
+<p>Follow <a href="https://twitter.com/nttwconf">@nttwconf</a> for news and updates about No Time to Wait.</p>
 
 <h2>When</h2>
-<p>(To Be Announced)</p>
+<p>Thursday, October 25, 2018 and Friday, October 26, 2018</p>
 
+<p>Opening: 09:00<br />
+Closing: 17:00</p>
 
 <h2>Where</h2>
 <p><a href="http://www.bfi.org.uk/">British Film Institute</a>, Belvedere Rd, South Bank, London SE1 8XT, UK</p>
 
-<p>Parts of the workshop will also be accessible remotely</p>
+<p><img src="http://www.bfi.org.uk/sites/bfi.org.uk/files/styles/full/public/image/bfi-southbank-2013-001-exterior-evening.jpg?itok=hkPPfTCN" width="500" alt="British Film Institute" /><br />
+<em>Photo Credit: <a href="http://www.bfi.org.uk/sites/bfi.org.uk/files/styles/full/public/image/bfi-southbank-2013-001-exterior-evening.jpg?itok=hkPPfTCN">British Film Institute</a></em></p>
 
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2483.385037797375!2d-0.11740564823185592!3d51.506151679535094!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x487604b7e7c78711%3A0x3427f18d546137bd!2sBFI+Southbank!5e0!3m2!1sen!2sus!4v1525800764082" width="600" height="450" frameborder="0" style="border:0" allowfullscreen=""></iframe>
 
-<h2>Registration:</h2>
+<p>Parts of the workshop will also be accessible remotely. More info soon.</p>
 
-<p>(To Be Announced)</p>
+<h2>Registration</h2>
 
+<p>Please fill out our <a href="http://bit.ly/nttw3signup">REGISTRATION FORM</a>.</p>
 
-<h2>Travel Grant</h2>
+<h2>Travel Grants</h2>
 
 <p>(To Be Announced)</p>
 
 <p>If you would like to sponsor additional travel grants, <a href="{{ path('ma_contact') }}">please contact us</a>.</p>
 
-
 <h2>Program:</h2>
 
-<p>(To Be Announced)</p>
-
+<p>(To Be Announced) A Call for Proposals shall be issued soon.</p>
```

Comment:
isn't Call for Proposals URL already available?

Reply:
Announce first, update with active CFP later?